backend/api/notifications.py

```diff
@@ -14,17 +16,22 @@ def add_notification(creator, userlist, operation, project_id='',
     date = time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime())
     for user in userlist:
         if user not in (creator):
-            g.notifications.insert_one({
-                '_id': str(uuid.uuid4()),
-                'creator': creator,
-                'user_id': user,
-                'project_id': project_id,
-                'operation': operation,
-                'date': date,
-                'reason': reason,
-                'active': 'true',
-                'saved_search_id': saved_search_id
-            })
+            user_object = g.user_datastore.get_user(user)
+            if user_object:
+                dict = user_object.to_dict()
```

Comment:
Not working as supposed to be.... either you should rename it or it will be in the master nevertheless... that "issue" is at several places